packages/dart_firebase_admin/lib/src/app/firebase_admin.dart

```diff
@@ -9,19 +9,46 @@ class FirebaseAdminApp {
   /// The [Credential] used to authenticate the Admin SDK.
   final Credential credential;
 
-  bool get isUsingEmulator => _isUsingEmulator;
-  var _isUsingEmulator = false;
+  bool get isUsingAuthEmulator => _isUsingAuthEmulator;
+
+  bool get isUsingFirestoreEmulator => _isUsingFirestoreEmulator;
+
+  bool get isUsingEmulator => _isUsingAuthEmulator || _isUsingFirestoreEmulator;
+
+  var _isUsingAuthEmulator = false;
+  var _isUsingFirestoreEmulator = false;
 
   @internal
   Uri authApiHost = Uri.https('identitytoolkit.googleapis.com', '/');
   @internal
   Uri firestoreApiHost = Uri.https('firestore.googleapis.com', '/');
 
-  /// Use the Firebase Emulator Suite to run the app locally.
-  void useEmulator() {
-    _isUsingEmulator = true;
-    authApiHost = Uri.http('127.0.0.1:9099', 'identitytoolkit.googleapis.com/');
-    firestoreApiHost = Uri.http('127.0.0.1:8080', '/');
+  /// Use the Firebase Emulator suite to run the app locally.
+  void useEmulator({
+    Emulator? authEmulator,
+    Emulator? firestoreEmulator,
+  }) {
+    useAuthEmulator(emulator: authEmulator);
+    useFirestoreEmulator(emulator: firestoreEmulator);
+  }
+
+  /// Use the Firebase Auth Emulator to run the app locally.
+  void useAuthEmulator({
+    Emulator? emulator
+  }) {
```

Comment:
Wouldn't a simpler API be this?

```dart
void useAuthEmulator([Uri? uri])
```

We can then remove that `Emulator` class.